report/custom.go

```diff
@@ -353,3 +328,8 @@ func (d *DiffCustomMetricSet) MetadataTable() string {
 func isInt(v float64) bool {
 	return v == float64(int64(v))
 }
+
+func convertFormat(v interface{}) string {
+	p := message.NewPrinter(language.Japanese)
```

Comment:
That's right.
The three-digit comma delimiter is not universal.
Therefore, we are unable to include this fix.

ref: https://en.wikipedia.org/wiki/Decimal_separator